pytest_tests/testsuites/network/test_config_changes.py

```diff
@@ -0,0 +1,151 @@
+from typing import Union
+
+import allure
+import pytest
+from cluster_test_base import ClusterTestBase
+from common import NEOFS_ADM_CONFIG_PATH, NEOFS_ADM_EXEC
+from neofs_testlib.cli import NeofsAdm
+from python_keywords.neofs_verbs import get_netmap_netinfo
+
+CONFIG_KEYS_MAPPING = {
+    "MaxObjectSize": "maximum_object_size",
+    "BasicIncomeRate": "storage_price",
+    "AuditFee": "audit_fee",
+    "EpochDuration": "epoch_duration",
+    "ContainerFee": "container_fee",
+    # "ContainerAliasFee": "",
```

Comment:
Maybe remove the comment?

Reply:
I am waiting for an answer to my description questions. I'm not sure if 'ContainerAliasFee' is not shown by design or if it is a bug. I will either uncomment it and skip as a bug or just remove completely, depends on an answer

Reply:
Leave it, but create an issue that depends on https://github.com/nspcc-dev/neofs-node/issues/2553 (which in turn depends on https://github.com/nspcc-dev/neofs-sdk-go/issues/514).

Reply:
what about problems 2 and 3 in the description? are they bugs as well, or I should tolerate this behavior?